src/main/java/com/noticemedan/mappr/model/util/ElementMutator.java

```diff
@@ -0,0 +1,67 @@
+package com.noticemedan.mappr.model.util;
+
+import com.noticemedan.mappr.model.map.Element;
+
+import java.awt.*;
+import java.awt.geom.Path2D;
+import java.awt.geom.PathIterator;
+import java.util.ArrayList;
+
+public class ElementMutator {
+	public static ArrayList<Element> determineRoadMultiplicity(Element road) {
+		ArrayList<Element> identicalRoads =  new ArrayList<>();
```

Comment:
Hele vejen igennem?

Reply:
Ja han er gået tilbage til the dark side.